packages/tests/tests/light-push/custom_pubsub.node.spec.ts

```diff
@@ -25,6 +25,8 @@ describe("Waku Light Push [node only] - custom pubsub topic", function () {
       nwaku,
       customPubSubTopic
     );
+
+    await nwaku.ensureSubscriptions([customPubSubTopic]);
```

Comment:
We have other tests using a custom pubsub topic, no?

Reply:
Yes but subscriptions is require only if you want to check the nwaku message cache.